routes/product.js

```diff
@@ -1,22 +1,15 @@
 const express = require('express');
 const router = express.Router();
-const products = require('../src/data/products');
-const categories = require('../src/data/categories');
+const service = require('../services/dataService');
 
+/*
+ * This method gets the id of the product from the req params.
+ * It gets the combinedProductMap and returns the product of that id;
+ * Pretty Simple :)
+ */
 router.get('/:id', function(req, res, next) {
```

Comment:
Put this in `products.js` instead, since this is products related. No need to place the endpoint in a separate file.